package.json

```diff
@@ -41,7 +41,6 @@
     "playwright:install": "playwright install --with-deps",
     "postinstall": "remix-esbuild-override",
     "prepack": "npmignore --auto",
-    "prepare": "lefthook install 1>/dev/null 2>/dev/null || true",
```

Comment:
We still need this, it's to ensure that anyone pulling out a project also installs lefthook. Can't leave it out